### ISSUE TITLE
Active segments returned are not aligned to requested start / stop time

### ISSUE THREAD

**Issue by eagoetz** [duncanmmacleod/dqsegdb2#38]
Example:
```python3
from dqsegdb2.query import query_segments
foo = query_segments("H1:DMT-ANALYSIS_READY", 1266969618, 1267056018)
```

I expect that the start of the only active segment should be 1266969618 and end at 1267056018 since the IFO was locked in low noise observing at that time, but what I find is:
```
foo['active']
[segment(1266969616.0, 1267056064.0)]
```

I suggest this unexpected behaviour be changed so that the active segment is aligned with the requested start stop times.

**Comment by duncanmmacleod** [comment]
@eagoetz, this was fixed in 17649016acb3deea6b763008e814d50d16b95c7b which was released as part of 1.1.0.

**Comment by eagoetz** [comment]
@duncanmmacleod Ok, thanks. I guess it has not made it into `igwn-py39` yet. Thanks for checking and having already fixed it.
